website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -1,13 +1,11 @@
 ---
 layout: docs
-page_title: Connect - Service Registration
+page_title: Service Mesh Proxy: Configuration
```

Comment:
I don't like how clunky the title I suggested is, but it's a more accurate reflection of what is going on (and how it's different from the alternative).
```suggestion
page_title: Service Mesh Proxy Registration: Separate from Service Registration
```